src/mock-query.coffee

```diff
@@ -1,4 +1,5 @@
 mongoose = require 'mongoose'
+mongoose.Promise = require('q').Promise
```

Comment:
I feel like this should be configurable. For instance, if we pull this into wafflebot, we would want it to be native promises.